File: php/awqat/Content.php
<?php

declare(strict_types=1);

namespace Awqat;

/** DailyContent: günün Ayet / Hadis / Dua içeriği (parametresiz). */
class Content
{
    public static function dailyContent(Client $c)
    {
        return $c->getJson('/api/DailyContent');
    }
}

File: go/awqat/content.go
package awqat

import "context"

// DailyContent; günün Ayet / Hadis / Dua içeriğini getirir (parametresiz).
// GET /api/DailyContent
func (c *Client) DailyContent(ctx context.Context) (DailyContent, error) {
	return getJSON[DailyContent](ctx, c, "/api/DailyContent")
}
